src/mutation.rs

```diff
@@ -0,0 +1,251 @@
+use async_graphql::{Context, Error, Object, Result};
+use bson::Bson;
+use bson::Uuid;
+use mongodb::{
+    bson::{doc, DateTime},
+    Collection, Database,
+};
+
+use crate::product_variant::ProductVariant;
+use crate::query::query_user;
+use crate::user::User;
+use crate::{
+    mutation_input_structs::{AddReviewInput, UpdateReviewInput},
+    query::query_review,
+    review::Review,
+};
+
+/// Describes GraphQL review mutations.
+pub struct Mutation;
+
+#[Object]
+impl Mutation {
+    /// Entity resolver for user of specific id.
+    #[graphql(entity)]
+    async fn user_entity_resolver<'a>(
```

Comment:
having the entity resolver in the mutation thing looks a bit strange to me

Reply:
in particular as there is also a user_entity_resolver in query
do you need both?

Reply:
Yes, this is now unused code. Will fix